dbt_server/services/task_service.py

```diff
@@ -19,8 +19,9 @@ def run_task(task_name, task_id, args, db):
     log_path = filesystem_service.get_path(args.state_id, task_id, "logs.stdout")
 
     log_manager = LogManager(log_path)
-    log_manager.setup_handlers()
 
+    # TODO: Structured logging doesn't have the concept of custom log lines like this,
+    # need to follow up with core about a way to do this
```

Comment:
As discussed in Slack, this could potentially use the "adapter" endpoint in our logging interface, which lets you fire log events using stdlib-style methods:
```
from dbt.events import AdapterLogger
logger = AdapterLogger("Server")
```
This would then replace `from dbt_server.logging import GLOBAL_LOGGER` up above.